api/src/main/java/org/openmrs/module/fhir/api/strategies/condition/GenericConditionStrategy.java

```diff
@@ -42,4 +42,13 @@
 	 * @return Created FHIRCondition
 	 */
 	Condition createFHIRCondition(Condition condition);
+
+	/**
+	 * Update Condition
+	 *
+	 * @param condition FHIR representation of the {@link}Condition to be updated
+	 * @param uuid uuid of the condition to be updated
```

Comment:
Why do you pass in both a condition and uuid?

Reply:
To make sure that the intended condition is updated.

Reply:
Doesn't the to be updated condition already have a uuid?

Reply:
It has.